app/controllers/teams_controller.rb

```diff
@@ -6,11 +6,19 @@ class TeamsController < ApplicationController
   load_and_authorize_resource except: [:index, :show]
 
   def index
```

Comment:
This looks a bit confusing to me 🙈 
What's the point in 3 times overriding the `teams` instance variable`? This is going to end up in multiple - redundant after all - calls to the database, which is usually not what you want. Can you think of a way to structure this nicer and get rid of redundant DB calls?

Reply:
@klappradla done :)